backend/uclapi/oauth/views.py

```diff
@@ -50,17 +50,10 @@ def authorise(request):
         response.status_code = 400
         return response
 
-    if app.callback_url is None or app.callback_url.strip() == "":
+    if app.callback_url is None:
         response = PrettyJsonResponse({
             "ok": False,
-            "error": (
-                "This app does not have a callback URL set. "
-                "If you are the developer of this app, "
-                "please ensure you have set a valid callback "
-                "URL for your application in the Dashboard. "
-                "If you are a user, please contact the app's "
-                "developer to rectify this."
-            )
+            "error": "No callback URL set for this app."
```

Comment:
For some reason this loses the nicer error message from the other PR :(

Reply:
yeh the merging seems to randomly throw away my commits I can't for the life of me figure it out

Reply:
I blame @torvalds himself 😉